packages/p2p/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "deriv-p2p",
+  "version": "1.0.4",
+  "description": "p2p cashier",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"No test specified\"",
+    "build": "rimraf index.js && webpack --mode=production --optimize-minimize --progress",
+    "build:travis": "rimraf index.js && webpack --mode=production --optimize-minimize",
+    "serve": "rimraf index.js && cross-env BUILD_MODE='serve' webpack --mode=production --progress --watch",
+    "start": "rimraf index.js && npm run test && webpack --mode=production --progress --watch",
+    "translate": "sh ./scripts/update-translations.sh",
```

Comment:
can be removed

Reply:
@oskar-binary Can you please specify which ones? Because `serve`, `start`, and `build:travis` are needed.

Reply:
i removed, the translate